### PR TITLE
LPS-72391 Drag and drop image in web content

### PR DESCRIPTION
**Pull request by jkappler**

### CODE REVIEW

File: journal-web/src/main/java/com/liferay/journal/web/internal/upload/TempImageJournalUploadFileEntryHandler.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
+/**
```
Comment (ealonso) [inline journal-web/src/main/java/com/liferay/journal/web/internal/upload/TempImageJournalUploadFileEntryHandler.java]: Why we don't use the ImageJournalUploadFileEntryHandler?

File: journal-service/src/main/java/com/liferay/journal/util/impl/JournalArticleAttachmentContentUpdater.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
+/**
```
Comment (ealonso) [inline journal-service/src/main/java/com/liferay/journal/util/impl/JournalArticleAttachmentContentUpdater.java]: Can we extract this code to a generic class to reuse on blogs and here?

File: journal-service/src/main/java/com/liferay/journal/util/impl/JournalArticleAttachmentFileEntryUtil.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
+/**
```
Comment (ealonso) [inline journal-service/src/main/java/com/liferay/journal/util/impl/JournalArticleAttachmentFileEntryUtil.java]: Can we reuse this code on JournalArticleLocalServiceImpl?

**Comment by ealonso** [comment]
Hey @jkappler,

Could you please take a look to the comments and resend?

Thanks!!!

**Comment by jkappler** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**